src/options.cpp

```diff
@@ -827,6 +827,10 @@ void initOptions()
                                    _("Adjust the volume of the music being played in the background."),
                                    0, 200, 100, COPT_CURSES_HIDE
                                   );
+    OPTIONS["SOUND_VOLUME"] = cOpt("graphics", _("Sound Volume"),
+                                   _("Adjust the volume of sound effects being played by the game."),
```

Comment:
Yes, I wasn't sure whether sound effect volume would be too long, but I agree now it's more accurate. Maybe SFX Volume? But then some people may not know what it means.